media/css/cms/flare-theme.css

```diff
@@ -68,11 +68,11 @@ runtime conditions. Design tokens live in flare-tokens.css.
     --fl-theme-code-surface: var(--token-neutrals-grey-3);
 
     /* Typography variables */
-    --fl-theme-font-family-headline: var(--token-font-family-metropolis);
+    --fl-theme-font-family-headline: var(--token-font-family-mozilla-headline);
```

Comment:
Changing these font family variables here is also changing which font families are used on the Whats New Page